libraries/rush-lib/src/logic/Autoinstaller.ts

```diff
@@ -109,10 +109,13 @@ export class Autoinstaller {
       rushJsonFolder: this._rushConfiguration.rushJsonFolder
     });
 
-    if (!lastInstallFlag.isValid() || lock.dirtyWhenAcquired) {
-      // Example: ../common/autoinstallers/my-task/node_modules
-      const nodeModulesFolder: string = path.join(autoinstallerFullPath, 'node_modules');
+    // Example: ../common/autoinstallers/my-task/node_modules
+    const nodeModulesFolder: string = path.join(autoinstallerFullPath, RushConstants.nodeModulesFolderName);
+    const isLastInstallFlagDirty: boolean =
+      !lastInstallFlag.isValid() ||
+      !FileSystem.exists(path.join(nodeModulesFolder, 'rush-autoinstaller.log'));
```

Comment:
```suggestion
      !FileSystem.exists(`${nodeModulesFolder}/${rush-autoinstaller.log}`);
```

Reply:
`path.join` is somewhat slow, and Windows doesn't care if you use forward or backslashes, so in cases where the parameters are well-known, it's faster to just concatenate the parts of the path with `/`es